pom.xml

```diff
@@ -89,9 +92,49 @@
                 <version>1.6.3</version>
                 <extensions>true</extensions>
                 <configuration>
-                    <serverId>ossrh</serverId>
+                    <serverId>sonatype-nexus-staging</serverId>
```

Comment:
Im deploying to staging repository, and they promote it to production.. i can push to production directly